source/lib.civet

```diff
@@ -1061,6 +1082,26 @@ function removeParentPointers(node): void {
   }
 }
 
+/**
+ * Replace `child` with `replacement` in `node.children`.
```

Comment:
This seems to be modifying expressions, not children. Update comment, and maybe function name?